fair_drop/suspicious.py

```diff
@@ -0,0 +1,180 @@
+# See https://github.com/convex-labs/honestnft-shenanigans/issues/86 for more details
+# Given a collection of NFTs on OpenSea, detect suspicious NFTs
+import time
+from multiprocessing import Pool
+from argparse import ArgumentParser
+import logging
+
+import cloudscraper
+import pandas as pd
+from bs4 import BeautifulSoup
+from requests.adapters import HTTPAdapter, Retry
+
+
+logging.basicConfig(level=logging.INFO)
+
+parser = ArgumentParser()
+parser.add_argument(
+    "-c",
+    "--collection",
+    dest="collection_address",
+    help="Address of the NFT collection",
+    metavar="COLLECTION",
+    required=True,
+)
+parser.add_argument(
+    "-r",
+    "--retries",
+    dest="retries",
+    help="Number of retry attempts",
+    epilog="See https://urllib3.readthedocs.io/en/stable/reference/urllib3.util.html",
+    metavar="RETRIES",
+    required=False,
+    default=3,
+)
+parser.add_argument(
+    "--backoff",
+    dest="backoff",
+    help="Retries backoff parameter for failed requests",
+    epilog="See https://urllib3.readthedocs.io/en/stable/reference/urllib3.util.html",
+    metavar="RETRIES",
+    required=False,
+    default=3,
+)
+parser.add_argument(
+    "-b",
+    "--batch-size",
+    dest="batch_size",
+    help="Batch size of NFT URLs to be processed in parallell",
+    metavar="RETRIES",
+    required=False,
+    default=50,
+)
```

Comment:
`$ py3 fair_drop/suspicious.py -c 0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d --batch-size 50`
Gives me this error:
```
INFO:root:Collection contains 9999 NFTs
INFO:root:New collection to scrape. No cache detected.
INFO:root:Found 0 NFTs in collection cache
INFO:root:Scraping a list of 9999 NFTs
Traceback (most recent call last):
  File "/home/xxx/honestnft-shenanigans/fair_drop/suspicious.py", line 180, in <module>
    scrape_all_collection_suspicious_nfts(args.collection_address)
  File "/home/xxx/honestnft-shenanigans/fair_drop/suspicious.py", line 137, in scrape_all_collection_suspicious_nfts
    for i in range(0, len(collection_nfts_urls), args.batch_size)
TypeError: 'str' object cannot be interpreted as an integer

```

Reply:
Converted the CLI parameters to integer